### PR TITLE
abort run if user cancels (C-g) a scene/script selection.

### PR DESCRIPTION
**Pull request by rileyrg**

### CODE REVIEW

File: gdscript-godot.el [hunk 1]
```diff
@@ -102,20 +102,26 @@ When run with prefix argument, it offers extra debug options to choose from."
...
+      (gdscript-godot--run-command scene))))
 
 (defun gdscript-godot-run-current-scene-debug ()
   "Run the current script file in Godot Engine.
 
 When run with prefix argument, it offers extra debug options to choose from."
   (interactive)
-  (gdscript-godot--debug-options-handler debug-options
-    (gdscript-godot--run-command "-d" debug-options (gdscript-godot--select-scene))))
+  (let ((scene (gdscript-godot--select-scene current-prefix-arg))) ;; TODO - debug and force select using prefix. need to tweak this sometime.
```
Comment (NathanLovato) [inline gdscript-godot.el:114]: Please don't leave comments and, in particular, TODOs in the code. If a feature is unfinished or has a known bug, it should be addressed before merging. If a feature can be improve, then please open a new issue.

**Review by NathanLovato — CHANGES_REQUESTED** [review]
Could you remove the TODO note from the source code?

Also, may I ask what this feature is and how I can test it? Does it only work if you don't have projectile or ivy?

File: gdscript-godot.el (hunk 1 above)
Comment (rileyrg) [reply gdscript-godot.el:114]: Prior to this fix if you hit C-g (I use projectile) it still ran the default scene whereas it should abort.   

Re todo, yes I'll address that tomorrow. End of day here.

Comment (NathanLovato) [reply gdscript-godot.el:114]: Gotcha, thanks much!

**Review by NathanLovato — APPROVED** [review]

**Comment by NathanLovato** [comment]
Thanks!

**Pull request merged**